### ISSUE TITLE
AttributeError: module 'cv2.gapi' has no attribute 'wip'

### ISSUE THREAD

**Issue by ItsZer0here** [skvark/opencv-python#911]
### Expected behaviour

The script had to RUN to record frames

### Actual behaviour

Threw me the error `AttributeError: module 'cv2.gapi' has no attribute 'wip'`

### Steps to reproduce

1. import cv2
2. that's it

- example code: `import cv2`
- operating system: Windows 11 Pro 21H2
- architecture (e.g. x86): x64
- opencv-python version: 4.8.1.78 (contrib)

##### Issue submission checklist

 - [x] This is not a generic OpenCV usage question (looking for help for coding, other usage questions, homework etc.)
   
 - [x] I have read the README of this repository and understand that this repository provides only an automated build toolchain for OpenCV Python packages (there is no actual OpenCV code here)
   
 - [x] The issue is related to the build scripts in this repository, to the pre-built binaries or is a feature request (such as "please enable this additional dependency")
 - [x] I'm using the latest version of ``opencv-python``

**Comment by asmorkalov** [comment]
Most probably you have more than one instance of OpenCV-Python installed on your system with different sources. Please check `pip list` command.

**Comment by ItsZer0here** [comment]
> Most probably you have more than one instance of OpenCV-Python installed on your system with different sources. Please check `pip list` command.

Nope, I only have one OpenCV-Python version: `opencv-contrib-python` version `4.8.1.78`

**Comment by parmentelat** [comment]
FWIW
I ran into this too just now
I was able to get things back to working by simply updating all my pips

**Comment by StormsHalted** [comment]
I got the same error today:

`AttributeError: module 'cv2.gapi' has no attribute 'wip'`

Uninstalling open CV and reinstalling it might fix.
